src/main/java/org/opentripplanner/apis/gtfs/GraphQLScalars.java

```diff
@@ -50,6 +54,34 @@ public String parseLiteral(Object input) {
     )
     .build();
 
+  public static GraphQLScalarType offsetDateTimeScalar = GraphQLScalarType
+    .newScalar()
+    .name("OffsetDateTime")
+    .coercing(
+      new Coercing<OffsetDateTime, String>() {
+        @Override
+        public String serialize(@Nonnull Object dataFetcherResult)
+          throws CoercingSerializeException {
+          if (dataFetcherResult instanceof ZonedDateTime zdt) {
+            return zdt.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+          } else if (dataFetcherResult instanceof OffsetDateTime odt) {
+            return odt.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+          } else return null;
+        }
+
+        @Override
+        public OffsetDateTime parseValue(Object input) throws CoercingParseValueException {
+          return null;
```

Comment:
This should be implemented, even if it is not used.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5660/commits/bf7dc55191a5cfcd24c853b703769120ccf51eaa